### ISSUE TITLE
vscode "source.organizeImports" breaks everything

### ISSUE THREAD

**Issue by sammosna** [sveltejs/language-tools#1703]
### Describe the bug

I have installed the new Svelte for vscode and enabled `source.organizeImports `. When I save, unused imports are removed but then all imports start flashing and vscode does not save the file

https://user-images.githubusercontent.com/6976083/197487968-2b358ad7-a5b0-4bd4-852a-1b81f55f70b3.mov

 

### Reproduction

Install Svelte for vscode and put this in your settings
```json
"editor.codeActionsOnSave": {
        "source.organizeImports": true,
},
```

### Expected behaviour

It should save the file and not flash

### System Info

- OS: MacOS 12.6 Monterey
- IDE: VSCode Insiders

### Which package is the issue about?

Svelte for VS Code extension

**Comment by dummdidumm** [comment]
Could you provide a copy-pasteable code snippet that reproduces this for you? Also, what else is running on save? Code format? If yes, through which extension?

**Comment by insilications** [comment]
I had the same problem. Currently disabling  "source.organizeImports"

**Comment by jasonlyu123** [comment]
This is likely having something to do with your other config. If you have the same problem, Please check if another config or extension is causing the trouble. Otherwise, we can't help you.

**Comment by ZizhengTai** [comment]
I'm also having this issue. What kind of output might be useful for debugging this? Does the Svelte VSCode extension output logs somewhere?

**Comment by jasonlyu123** [comment]
Fixed in VSCode 1.73.1
